modules/modal/react/spec/SSR.spec.tsx

```diff
@@ -0,0 +1,13 @@
+/**
+ * @jest-environment node
+ */
+import React from 'react';
+import {renderToString} from 'react-dom/server';
+import Modal from '../';
+
+describe('Modal', () => {
+  it('should render on a server without crashing', () => {
+    const ssrRender = () => renderToString(<Modal open={true} heading="test" />);
```

Comment:
Will we ever render an open `Modal` from the server? Does that even make sense? Typically modals are triggered by some type of client event.

Reply:
Probably not, I was just trying to properly test `ModalContent` since it references `document`. Could drop this prop if you think that makes more sense